app/assets/javascripts/miq_angular_application.js

```diff
@@ -5,6 +5,7 @@ ManageIQ.angular.app = angular.module('ManageIQ', [
 ]);
 miqHttpInject(ManageIQ.angular.app);
 
+ManageIQ.angular.rxSubject = new Rx.Subject();
```

Comment:
In the future, are you expecting to use this one rxSubject for non-toolbar things, or would we go with other instances?

I'm asking because `rxSubject` sounds rather too generic for a toolbar-specific thing..